components/commands/common/key_names.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_COMMANDS_COMMON_KEY_NAMES_H_
+#define BRAVE_COMPONENTS_COMMANDS_COMMON_KEY_NAMES_H_
+
+#include <string>
+#include <vector>
+
+#include "ui/events/event_constants.h"
+#include "ui/events/keycodes/keyboard_codes.h"
+
+namespace commands {
+std::string GetKeyName(ui::KeyboardCode code);
+std::vector<std::string> GetModifierName(ui::KeyEventFlags flags);
```

Comment:
I'm wondering if we can use Accelerator::GetShortcutTexts() instead of managing all the characters. There could be problems like
* We might have to split the shorcut text with +
* We might have to distinguish which one is modifiers - maybe we could consider names longer than 1 as modifiers.

Reply:
I just had a play with changing it over, and it's quite a bit worse - there are a bunch of missing keys, and a bunch of keys which are displayed wrong:

1. All function keys are missing
2. Special keys are mislabelled (`Keyboard back/forward` is `Arrow left/right`)
3. Cmd modifier doesn't seem to show up on Linux

I think longer term it might be good to try and fix the GetShortcutsText function (maybe upstream...) but for now this is strictly better :'(

Reply:
> (maybe upstream...)

Yeah, I don't want put you that tough way haha. Maybe we can wrap the GetShortcutText() and fix up some of the results later on.